scripts/clang-format/clang-format.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd ${PWD}/scripts/clang-format # assume current dir is arangodb (not error-proof)
+docker build -t clang-format .
+while [[ $PWD != '/' && ${PWD##*/} != 'arangodb' ]]; do cd ..; done # cd up to arangodb
```

Comment:
The assumption of having a directory named `arangodb` feels strange and doesn't work for me right now (and probably won't work for a lot of others).

Reply:
We may get away with something like `pushd ... popd` here. These seem to be supported by most of the relevant shells today.